### DISCUSSION TITLE
Is FaunaDB still planned for support?

### DISCUSSION THREAD

**Discussion by cysabi** [prisma/prisma#13241] [Q&A] [answered]
Hello, I apologize for the duplicate discussion but any news about supporting FaunaDB seems to be either stale or outdated.
I have seen the main issue regarding support (#2445) as well as the rust driver.

However, the issue has not had any activity by maintainers in a year, and the driver has not even had any commits since 3 years ago.

Is this still being planned for support? Or has it been dropped / on the back burner for more prioritized issues?

**Comment by floelhoeffel** [comment, marked as answer]
Hey @LeptoFlare,

Thanks for submitting this question! Long term we really want to support more database connectors so Prisma can be used with your stack of choice. 

However, building and maintaining connectors is very hard. Our teams just wrapped up two new connectors (MongoDB and CockroachDB) which were multi quarter long initiatives each. So now our priority is on improving support for the existing connectors. Unfortunately this means we don't have bandwidth to add more just yet.

I hope this explains our current thinking and I recommend to checkout our public roadmap to see which work is planned: https://www.prisma.io/docs/about/prisma/roadmap

Cheers,
Frederik, PM

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
